### PR TITLE
[DOC-3341] Update podman docker delegate install

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update podman docker delegate install

For reviewers, [a preview](https://649d9563ceee8f16f0c892b9--harness-developer.netlify.app/docs/platform/Delegates/install-delegates/overview#install-docker-delegate-using-podman) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]

File: docs/platform/2_Delegates/install-delegates/overview.md [hunk 1]
```diff
@@ -73,76 +73,25 @@ The legacy Kubernetes delegate, denoted `latest` container image tag, is used pr
...
+podman pull docker.io/harness/delegate:yy.mm.xxxxx
 
--e DELEGATE_TOKEN=YOUR_DELEGATE_TOKEN \
+podman run --restart=always --hostname="$(hostname -f | head -c 63)"
+-e DELEGATE_NAME=podman-delegate
+-e NEXT_GEN="true"
+-e DELEGATE_TYPE="DOCKER"
+-e ACCOUNT_ID=YOUR_ACCOUNT_ID
+-e DELEGATE_TOKEN=YOUR_DELEGATE_TOKEN
+-e LOG_STREAMING_SERVICE_URL=https://app.harness.io/gratis/log-service/
```
Comment (xingchi-jin) [inline docs/platform/2_Delegates/install-delegates/overview.md:93]: -e LOG_STREAMING_SERVICE_URL=YOUR_MANAGER_HOST_AND_PORT/log-service/
-e MANAGER_HOST_AND_PORT=YOUR_MANAGER_HOST_AND_PORT

File: docs/platform/2_Delegates/install-delegates/overview.md [hunk 2]
```diff
@@ -73,76 +73,25 @@ The legacy Kubernetes delegate, denoted `latest` container image tag, is used pr
...
 
-sudo podman run -d --restart=always --hostname="$(hostname -f | head -c 63)" \
+sudo apt-get -y install podman
 
--e ACCOUNT_ID=YOUR_ACCOUNT_ID \
+podman pull docker.io/harness/delegate:yy.mm.xxxxx
 
--e DELEGATE_TOKEN=YOUR_DELEGATE_TOKEN \
+podman run --restart=always --hostname="$(hostname -f | head -c 63)"
+-e DELEGATE_NAME=podman-delegate
```
Comment (xingchi-jin) [inline docs/platform/2_Delegates/install-delegates/overview.md:88]: -e DELEGATE_NAME=YOUR_DELEGATE_NAME

File: docs/platform/2_Delegates/install-delegates/overview.md [hunk 3]
```diff
@@ -73,76 +73,25 @@ The legacy Kubernetes delegate, denoted `latest` container image tag, is used pr
...
+sudo apt-get -y update
 
-sudo podman run -d --restart=always --hostname="$(hostname -f | head -c 63)" \
+sudo apt-get -y install podman
 
--e ACCOUNT_ID=YOUR_ACCOUNT_ID \
+podman pull docker.io/harness/delegate:yy.mm.xxxxx
 
--e DELEGATE_TOKEN=YOUR_DELEGATE_TOKEN \
+podman run --restart=always --hostname="$(hostname -f | head -c 63)"
```
Comment (xingchi-jin) [inline docs/platform/2_Delegates/install-delegates/overview.md:87]: --hostname="$(hostname -f)

File: docs/platform/2_Delegates/install-delegates/overview.md (hunk 2 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/install-delegates/overview.md:88]: updated

File: docs/platform/2_Delegates/install-delegates/overview.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/install-delegates/overview.md:93]: updated

File: docs/platform/2_Delegates/install-delegates/overview.md (hunk 3 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/install-delegates/overview.md]: updated

**Review by xingchi-jin — DISMISSED** [review]

File: docs/platform/2_Delegates/install-delegates/overview.md [hunk 4]
```diff
@@ -73,76 +73,34 @@ The legacy Kubernetes delegate, denoted `latest` container image tag, is used pr
...
--e CF_CLI7_PATH= \
-
--e KUSTOMIZE_PATH= \
-
--e OC_PATH= \
+podman ps
+```
 
--e KUBECTL_PATH= \
+Replace the `PUT_YOUR_MANAGER_HOST_AND_PORT_HERE` variable with the Harness Manager Endpoint noted below. For Harness SaaS accounts, you can find your Harness Cluster Location on the **Account Overview** page under the **Account Settings** section of the left navigation. For Harness CDCE, the endpoint varies based on the Docker vs. Helm installation options.
```
Comment (aimurphy) [inline docs/platform/2_Delegates/install-delegates/overview.md:98]:
```suggestion
Replace the `PUT_YOUR_MANAGER_HOST_AND_PORT_HERE` value with the Harness Manager Endpoint noted below. For Harness SaaS accounts, you can find your Harness Cluster Location on the **Account Overview** page under the **Account Settings** section of the left navigation. For Harness CDCE, the endpoint varies based on the Docker vs. Helm installation options.
```

Comment (aimurphy) [reply docs/platform/2_Delegates/install-delegates/overview.md:98]: Just a technicality that you're replacing the value of a variable. Feel free to ignore if this breaks the established vocabulary in your docs.

**Review by aimurphy — DISMISSED** [review]
one optional suggestion

**Pull request merged**